Resources/Prototypes/_NF/Entities/Markers/Spawners/Conditional/mobs_hostile_aberrant_flesh.yml

```diff
@@ -0,0 +1,18 @@
+- type: entity
+  name: aberrant flesh miner spawner
+  id: SpawnMobAberrantFleshMinerExpeditions
+  parent: MarkerBase
+  noSpawn: true
+  suffix: AI, Hostile, RND
+  components:
+  - type: Sprite
+    layers:
+    - state: red
+    - sprite: _NF/Mobs/Aliens/flesh.rsi
+      state: lover
+    - state: ai
+  - type: RandomSpawner
+    prototypes:
+    - MobFleshAssimilatedMiner
+    chance: 1
+    offset: 0.0
```

Comment:
Only used in the test spawner, right?  Should they be grouped?  If not, since this has a chance of 1, should it really have the "RND" tag?  Is that supposed to denote random chance (which this doesn't really have), or that it's using a RandomSpawner at all?

Reply:
Ah, yes, should be conditional spawner, not random

Reply:
Actually, will I'll remove this spawner and testing sack.